### DISCUSSION TITLE
Snapshot Voting Strategies while minting via Manifold.

### DISCUSSION THREAD

**Discussion by dchannn04** [snapshot-labs/snapshot#3545] [General]
I am looking to mint my NFTs via Manifold. However, I want to use this Snapshot voting strategy for my NFTs - https://snapshot.org/#/strategy/erc721-with-metadata. I was wondering if minting on Manifold would be compatible with Snapshot. If so, is there anything specific I need to do during the mint process to make sure my NFTs have the right metadata structure to follow this strategy?

**Comment by ChaituVR** [comment]
Hey @dchannn04 Make sure there is `tokenURI` in your contract and 
format of your metadata should be like:
```
[
    {
        [tokenID]: [value]
    }
]
```
Refer the documentation that is available here
https://snapshot.org/#/strategy/erc721-with-metadata

**Reply by dchannn04** [reply]
@ChaituVR how do I ensure that there is tokenURI in my contract?

**Reply by ChaituVR** [reply]
Refer https://eips.ethereum.org/EIPS/eip-721
